src/components/rating/CdrRating.vue

```diff
@@ -132,24 +139,23 @@ export default defineComponent({
   >
     <div :class="style['cdr-rating__ratings']">
 
-      <span
+      <CdrStar100
         v-for="star in Array(whole).keys()"
-        :class="mapClasses(style, 'cdr-rating__icon', 'cdr-rating__100')"
+        :size="size"
         :key="`rating-whole-${star}`"
         aria-hidden="true"
       />
-      <span
+      <component
         v-if="remainder !== '00'"
-        :class="mapClasses(style, 'cdr-rating__icon', `cdr-rating__${remainder}`)"
+        :is="'CdrStar' + remainder"
```

Comment:
Let's stick to string interpolation for consistency

`:is="`CdrStar${remainder}`"`

Reply:
No action needed though, I'll take care of this when merging the features

Reply:
Sounds good